### PR TITLE
Admin panel and change password [JupyterHub]

### PR DESCRIPTION
**Pull request by georgejhunt**
Fixes bug: #2919 and #2918  

Tested on clean lite RaspiOS 5/7/21 with min IIAB on Rpi4 hardware

### CODE REVIEW

**Comment by holta** [comment]
Wow.  This is tremendous news.

See also @georgejhunt's upstream PR's:

- https://github.com/jupyterhub/firstuseauthenticator/pull/37
- https://github.com/jupyterhub/firstuseauthenticator/pull/38

File: roles/jupyterhub/templates/patch_await.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
+# add await to asyncio change password function
+
+cat /opt/iiab/jupyterhub/lib/python3.7/site-packages/firstuseauthenticator/firstuseauthenticator.py |grep 'await self.render'
```
Comment (holta) [inline roles/jupyterhub/templates/patch_await.sh]: @georgejhunt doesn't `python3.7` need to be softcoded using {{ python_ver }} so this works on different versions of different OS's?

Ideally with `patch_await.sh.j2` if so?

Comment (holta) [reply roles/jupyterhub/templates/patch_await.sh]: If this is true (I don't know, I'm asking?) then Line 7 would need it too of course.

**Comment by georgejhunt** [comment]
This has been tested.  Please explain exactly what you think should be
changed.
The "cat . . ." line you quoted is only to test whether the patch has been
applied, in order to only apply it once

**Comment by holta** [comment]
@georgejhunt I'm asking if this path is correct, across all the different OS's that IIAB supports:

```
/opt/iiab/jupyterhub/lib/python3.7/site-packages/firstuseauthenticator/firstuseauthenticator.py
```

It doesn't look correct.  But I'm just trying to understand if `python3.7` needs to be softcoded.  So that this works on different OS's, each of which brings a different version of Python.

**Pull request merged**